src/libraries/System.Collections.Concurrent/tests/BlockingCollectionTests.cs

```diff
@@ -16,7 +16,7 @@ namespace System.Collections.Concurrent.Tests
     /// <summary>The class that contains the unit tests of the BlockingCollection.</summary>
     public class BlockingCollectionTests
     {
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
+        [Fact]
         public static void TestBasicScenarios()
```

Comment:
and this one. I'll stop commenting on these since I'm wondering how they passed for you :)

Reply:
Ah I see. It's also failing for me but I specifically grepped for `WASM: System.PlatformNotSupportedException : Cannot wait on monitors on this runtime.` and `WASM: System.PlatformNotSupportedException : Cannot start threads on this runtime.` and didn't think to look for variations where it still has `Cannot start threads on this runtime` like `WASM: ---- System.PlatformNotSupportedException : Cannot start threads on this runtime.` where there are dashes.

Reply:
I saw `CompositeChangeTokenTest` and `DefaultHttpClientFactory` in my wasm testing output: [wasm_test_output_original.txt.zip](https://github.com/dotnet/runtime/files/4799135/wasm_test_output_original.txt.zip), but didn't find this class and method. Not sure how it wasn't ran or how many others weren't run with `./build.sh --subset libs.tests -t --arch wasm --os Browser -c release`

Reply:
yeah from what I've seen the test suite crashes sometimes before getting to this method